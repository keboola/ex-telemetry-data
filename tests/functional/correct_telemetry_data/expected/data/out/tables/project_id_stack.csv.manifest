{"columns":["PROJECT_ID","PROJECT_STACK","sample_data"],"metadata":[{"key":"KBC.name","value":"project_id_stack"}],"column_metadata":{"PROJECT_ID":[{"key":"KBC.datatype.type","value":"varchar"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.datatype.length","value":"512"}],"PROJECT_STACK":[{"key":"KBC.datatype.type","value":"varchar"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.datatype.length","value":"512"}],"sample_data":[{"key":"KBC.datatype.type","value":"varchar"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.datatype.length","value":"512"}]}}