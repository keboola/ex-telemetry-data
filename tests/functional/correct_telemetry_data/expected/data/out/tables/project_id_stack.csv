"2","connection.north-europe.azure.keboola.com","test 7"
"2","connection.north-europe.azure.keboola.com","test 8"
